frontend/src/index.jade

```diff
@@ -199,6 +196,10 @@ html
 
     vuetemplate#v_authorship
       #authorship
+        // TODO: remove in #524
+        .toolbar(v-if="info.totalCommits > 0")
+            a(v-if="activeFileCount === 0", v-on:click="expandAll(true)") Expand all
```

Comment:
![Untitled](https://user-images.githubusercontent.com/18352498/54219855-2e0c9e80-452b-11e9-99d3-469577c6fa26.png)

Seems like this `activeFileCount` variable hasn't been refactored to latest master?